play/PlayerRules/CharacterCreation/03_Feats.txt

```diff
@@ -208,6 +208,15 @@ Prerequisites: None
 Somehow you always aim at the right location, at the right time. 
 +10% Crit chance, +5% Extra-Crit chance.
 
+Punch Drunk:
+Prerequisite: Fort > 3
+Take a +5% to miss on all attacks when not drunk. 
+Instead of taking a penalty to accuracy when using melee attacks when 
+intoxicated, you get a -5% to miss when buzzed and a -15% to miss when
+hammered. These bonuses only apply to melee attacks. You also get a 
++10% on rolls to avoid blacking out.  
+

```

Comment:
I think this should be buffed. It feels to me like you are in net equilibrium after this feat, ie., not gaining anything (especially since 'getting hammered' offers across the board maluses)
I recommend changing this:
>Take a +5% to miss on all attacks when not drunk. 

to "Take a +5% to miss on all _ranged_ attacks when not drunk."
or "Take a +5% to miss on all _melee_ attacks when not drunk."